server/athenian/api/align/queries/metrics.py

```diff
@@ -42,37 +53,59 @@
 
 @query.field("metricsCurrentValues")
 @sentry_span
-async def resolve_metrics_current_values(obj: Any,
-                                         info: GraphQLResolveInfo,
-                                         accountId: int,
-                                         params: Mapping[str, Any]) -> Any:
+async def resolve_metrics_current_values(
+    obj: Any, info: GraphQLResolveInfo, accountId: int, params: Mapping[str, Any]
```

Comment:
The trailing comma is missing. If black doesn't add it on purpose and unconfigurable, let's fork and hack it.

Reply:
for black trailing comma is mandatory when list is on multiple lines